src/adapters/bidderFactory.js

```diff
@@ -0,0 +1,240 @@
+import Adapter from 'src/adapter';
+import bidmanager from 'src/bidmanager';
+import bidfactory from 'src/bidfactory';
+import { STATUS } from 'src/constants';
+
+import { ajax, GET, POST } from 'src/ajax';
+import { logWarn, logError, parseQueryStringParameters, delayExecution } from 'src/utils';
+
+/**
+ * This file aims to support Adapters during the Prebid 0.x -> 1.x transition.
+ *
+ * Prebid 1.x and Prebid 0.x will be in separate branches--perhaps for a long time.
+ * This function defines an API for adapter construction which is compatible with both versions.
+ * Adapters which use it can maintain their code in master, and only this file will need to change
+ * in the 1.x branch.
+ *
+ * Typical usage looks something like:
+ *
+ * const adapter = newBidder({
+ *   code: 'myBidderCode',
+ *   areParamsValid: function(paramsObject) { return true/false },
+ *   buildRequests: function(bidRequests) { return some ServerRequest(s) },
+ *   interpretResponse: function(oneServerResponse) { return some Bids, or throw an error. }
+ * });
+ */
+
+/**
+ * This function aims to minimize the Adapter-specific code for "single request" Bidders. That is, if your adapter
+ * only makes a single request to the server for bids, regardless of how many bid requests or ad units are in
+ * the Auction, then this function is for you.
+ *
+ * @param {BidderSpec} spec An object containing the bare-bones functions we need to make a Bidder.
+ */
+export default function newBidder(spec) {
+  return Object.assign(new Adapter(spec.code), {
+    callBids: function(bidsRequest) {
+      if (!bidsRequest.bids || !bidsRequest.bids.filter) {
```

Comment:
This looks like an array check.  Would be more clear as `if (!Array.isArray(bidsRequest.bids) ) {`